.pa11y-local.json

```diff
@@ -0,0 +1,31 @@
+{
+    "urls": [
+        {
+          "url": "http://0.0.0.0:8000/report",
+          "actions": [
+            "wait for element #id_0-contact_phone to be visible"
+          ],
+          "screenCapture": "pa11y-screenshots/page1.png"
```

Comment:
We loved the screenshots feature, so awesome! Will save some time taking screenshots when doing local front-end development.

Reply:
I didn't even think of that use- great point!